app/src/main/java/org/oppia/android/app/application/AbstractOppiaApplication.kt

```diff
@@ -47,6 +50,8 @@ abstract class AbstractOppiaApplication(
     // TODO(#4751): Re-enable WorkManager for S+.
     if (Build.VERSION.SDK_INT < Build.VERSION_CODES.S) {
       FirebaseApp.initializeApp(applicationContext)
+      Firebase.firestore
```

Comment:
I don't know enough Kotlin, but I'm a bit surprised that these lines don't seem to be calling anything (shouldn't they have a "()" after them if they are functions)?

If they aren't functions then what do these lines do?

Reply:
In Kotlin getters use direct access syntax like this, so this returns the FirebaseFirestore instance of the default FirebaseApp.

Reply:
I suppose this means that I can call them directly at usage site instead of globally in this init block.

Reply:
But where is the returned value being stored? I don't see any "val ... = ..." statement.

Reply:
I have moved these getters to their call site where they are properly assigned to variables.